chia/rpc/wallet_rpc_api.py

```diff
@@ -2282,6 +2284,7 @@ async def did_find_lost_did(self, request: Dict[str, Any]) -> EndpointResult:
         if curried_args is None:
             return {"success": False, "error": "The coin is not a DID."}
         p2_puzzle, recovery_list_hash, num_verification, singleton_struct, metadata = curried_args
+        assert recovery_list_hash.atom is not None
```

Comment:
Seems like below we're doing `.as_atom()` which I think is not `Optional` so do we forsure need this?